observability/readme.md

```diff
@@ -26,6 +26,7 @@ Building observable systems enables development teams at CSE to measure how well
 1. [Logging](pillars/logging.md)
 2. [Tracing](pillars/tracing.md)
 3. [Metrics](pillars/metrics.md)
+4. [Profiling](pillars/profiling.md)
```

Comment:
There's some really good content here. I would argue that across the industry (AFAIK) Profiling has not been mentioned so far as a pillar. I can definitely see the value of having this entry in the playbook, just not sure as a pillar of observability.  

Keen to hear others opinions...

Reply:
That's a fair point! I've used it pretty extensively in a past role as an SRE, when we had it enabled on all of our services as continuous profiling. It was a fantastic tool to correlate any crashes with load spikes, and then to see if there was any memory leak (especially for htose smaller leaks that might only show up under high load)


Anyways, happy to remove it as a pillar, and link to it elsewhere. I'll wait till  end of week to see if anybody else wants to offer an opinion in favor of keeping it here, before making that update.